### ISSUE TITLE
Error: this.player.loadVideoById is not a function

### ISSUE THREAD

**Issue by edouard-lopez** [boris-graeff/LaBanane#4]
- **context:** Got the following error in Firefox `43.0b9`, works fine on Chromium.
- **behavior:** Can't play music from youtube or soundcloud.
- **question:** can you reproduce?
  
  ```
  Error: this.player.loadVideoById is not a function
  service.loadSong@http://labanane.no-ip.info/dist/app.js:499:21585
  $scope.play@http://labanane.no-ip.info/dist/app.js:499:4403
  $scope.play@http://labanane.no-ip.info/dist/app.js:499:4633
  anonymous/fn@http://labanane.no-ip.info/dist/app.js line 212 > Function:2:200
  @http://labanane.no-ip.info/dist/app.js:322:160
  kf/this.$get</m.prototype.$eval@http://labanane.no-ip.info/dist/app.js:132:354
  kf/this.$get</m.prototype.$apply@http://labanane.no-ip.info/dist/app.js:133:47
  @http://labanane.no-ip.info/dist/app.js:322:140
  Hf/c@http://labanane.no-ip.info/dist/app.js:35:124
   app.js:107:50
  
  Error: this.player.pauseVideo is not a function
  service.pause@http://labanane.no-ip.info/dist/app.js:499:21757
  $scope.pause@http://labanane.no-ip.info/dist/app.js:499:4682
  $scope.play@http://labanane.no-ip.info/dist/app.js:499:4120
  anonymous/fn@http://labanane.no-ip.info/dist/app.js line 212 > Function:2:278
  @http://labanane.no-ip.info/dist/app.js:322:160
  kf/this.$get</m.prototype.$eval@http://labanane.no-ip.info/dist/app.js:132:354
  kf/this.$get</m.prototype.$apply@http://labanane.no-ip.info/dist/app.js:133:47
  @http://labanane.no-ip.info/dist/app.js:322:140
  Hf/c@http://labanane.no-ip.info/dist/app.js:35:124
   app.js:107:50
  ```

**Comment by boris-graeff** [comment]
Fixed by f832cbc88ab766865aab7fcd413b7aa2e7958fef :)

**Comment by edouard-lopez** [comment]
:+1:
